pkg/controller/controllerutil/controllerutil_test.go

```diff
@@ -446,6 +446,18 @@ var _ = Describe("Controllerutil", func() {
 				Expect(deploy.ObjectMeta.GetFinalizers()).To(Equal([]string{}))
 			})
 		})
+
```

Comment:
Can you also add a test for `ContainsFinalizerWithError`, similar to the ones in lines 413-423?

Reply:
@joelanford , thanks, fixed